SAA Considerations for discussion (#1)
larseggert/bcp45bis: draft-eggert-bcp45bis.md

> # Security Considerations

Reviewer: I think that this section should be entitled "Moderation".  This document does not need security considerations and a separate security considerations section might say just that.

Reviewer 2: Good point. ~~This is a leftover from BCP45, where this content was in the Security Considerations, but that need not stay that way.~~ Actually, it's not - I added the Security Considerations section heading, so I will change it.

> A sergeant-at-arms (SAA) appointed by the IETF Chair is empowered to

Reviewer 3: Talking about terminology: When I mentioned this term to someone non-US outside of the IETF, that person was actually quite surprised by this term, given the role is supposed to explicitly not let the list get into a situation where "arms" are needed. Maybe this term is not as clear as it should be?

Author: FWIW the term is well established not just in the US but in most of the commonwealth nations. See [wiki](https://en.wikipedia.org/wiki/Serjeant-at-arms#Specific_countries).

Reviewer 3: And might still be confusing for all the other nations... just saying... don't have a problem myself.

Reviewer 2: FWIW, I was also somewhat surprised by the term when I started in the IETF. I wonder if it would be helpful to add a variant of the first sentence from the wikipedia page (e.g., "A  sergeant-at-arms is an officer appointed by a deliberative body to keep order during its meetings.") to the document, maybe with a link to the wikipedia page?

Reviewer 3: Yes, there were some/a few (?) people who raised that concern, however, at least one person also wanted an independent committee that is not appointed by the chair. So as long as the chair appoints the SAA, I don't see a big difference. I personally find it more important to make sure that everybody who serves as a SAA feels conformable with the position and for me that would also mean that they have someone they can approach and rely on in cause of problems, rather than having them taking full responsibility given this is a volunteer position.

Author: Sounds like a good idea to me!

restrict posting by a person, or of a thread, when the content is
inappropriate and represents a pattern of abuse. They are encouraged to
take into account the overall nature of the postings by an individual
and whether particular postings are an aberration or typical.

The sergeants-at-arms are intended to establish a self-moderation
function on the community, by the community. The IETF Chair therefore
should not appoint an SAA who is serving in a NomCom-appointed IETF
leadership position. If an SAA is selected for such a position, they
will step down as SAA.

Apart from appointing SAAs, the IETF Chair should divorce themselves
from the operation and management of the SAA team. This has been
in practice for a while, and the SAA team has
independently maintained definitions of abuse patterns {{SAA-UPC}} and

> operating procedures {{SAA-SOP}} for them. 

Reviewer 3: Doesn't the IETF chair still serve as a last resort if there is a conflict that can't be resolved by the SAA?

Author: I think this would be useful if there is a disagreement between the SAA team members; the IETF chair could help resolve it. But I am not sure if we need to add text describing this...

Reviewer 2: RFC3005 had:
>The IETF Chair, the IETF Executive Director, or a sergeant-at-arms
   appointed by the Chair is empowered to restrict posting by a person

I think the community has signalled that it doesn't want this power to remain with the IETF Chair, at least not directly. And I also think the changed role of the exec director makes them not very well suited for this.

Reviewer 3: Yes, there were some/a few (?) people who raised that concern, however, at least one person also wanted an independent committee that is not appointed by the chair. So as long as the chair appoints the SAA, I don't see a big difference. I personally find it more important to make sure that everybody who serves as a SAA feels conformable with the position and for me that would also mean that they have someone they can approach and rely on in cause of problems, rather than having them taking full responsibility given this is a volunteer position.

Author: That does resonate with me as the SAA. How about - 

```suggestion
Apart from appointing SAAs, the IETF Chair should divorce themselves
from the day-to-day operation and management of the SAA team. This 
has been in practice for a while, and the SAA team has independently 
maintained definitions of abuse patterns {{SAA-UPC}} and operating 
procedures {{SAA-SOP}} for them. The SAA team should reach out to 
the IETF chair for any conflict resolution in a timely manner.
```

Reviewer 2: Like it, but s/chair/Chair/ for consistency.

# Security Considerations

A sergeant-at-arms (SAA) appointed by the IETF Chair is empowered to
restrict posting by a person, or of a thread, when the content is
inappropriate and represents a pattern of abuse. They are encouraged to
take into account the overall nature of the postings by an individual
and whether particular postings are an aberration or typical.

The sergeants-at-arms are intended to establish a self-moderation
function on the community, by the community. The IETF Chair therefore
SHOULD NOT appoint an SAA who is serving in a NomCom-appointed IETF
leadership position. If an SAA is selected for such a position, they

> SHOULD consider stepping down as SAA.

Reviewer 4: ```suggestion
SHOULD step down as SAA.
```

We don't need to go all RFC 6919 here.

Reviewer 5: And imho we don't need to go all RFC 2119 either.

Reviewer: ```suggestion
will step down as SAA.
```

Apart from appointing SAAs, the IETF Chair SHOULD divorce themselves
from the operation and management of the SAA team. This has been
in practice for a while, and the SAA team has
independently maintained definitions of abuse patterns {{SAA-UPC}} and
operating procedures {{SAA-SOP}} for them. 

Because an SAA serves at the discretion of the IETF Chair - even if the
IETF Chair is not otherwise involved in the operation of the SAA team -

> complaints regarding any SAA decisions should be appealed to the IAB.

Reviewer 2: ```suggestion
SAA decisions should be appealed to the IAB.
```

Reviewer 6: Read literally, that's encouraging all complaints to be appealed. 

Do we need to say anything about the situation where an IAB member is involved in the dispute? Or the entire IAB?

Author: How about - 
```suggestion
any SAA decision could be appealed to the IAB. The IAB 
shall then review the situation and attempt to resolve it in 
a manner of its own choosing.
```

Do we need to say anything more?

# Security Considerations

A sergeant-at-arms (SAA) appointed by the IETF Chair is empowered to
restrict posting by a person, or of a thread, when the content is
inappropriate and represents a pattern of abuse. They are encouraged to
take into account the overall nature of the postings by an individual
and whether particular postings are an aberration or typical.

The sergeants-at-arms are intended to establish a self-moderation
function on the community, by the community. The IETF Chair therefore

> SHOULD NOT appoint an SAA who is serving in a NomCom-appointed IETF

Reviewer 2: ```suggestion
should not appoint an SAA who is serving in a NomCom-appointed IETF
```

leadership position. If an SAA is selected for such a position, they
will step down as SAA.

> Apart from appointing SAAs, the IETF Chair SHOULD divorce themselves

Reviewer 2: ```suggestion
Apart from appointing SAAs, the IETF Chair should divorce themselves
```
